src/libxtp/qmpackages/orca.cc

```diff
@@ -271,6 +271,28 @@ bool Orca::WriteInputFile(const Orbitals& orbitals) {
     WriteBackgroundCharges();
   }
 
+  // External Electric field
+  if (_settings.has_key("use_external_field")) {
+    if (_settings.get("use_external_field") == "true") {
+      if (_settings.has_key("externalfield")) {
+        tools::Tokenizer values(this->_settings.get("externalfield"), " ");
+        std::vector<std::string> field;
+        values.ToVector(field);
+        if(field.size() != 3){
+          throw std::runtime_error("Electric field does not have 3 values.");
+        }
+        inp_file << "%scf\n ";
+        inp_file << "  efield " << field[0] << ", " << field[1] << ", " << field[2] << "\n";
```

Comment:
Seems you found an undocumented Orca feature?

Reply:
Yeah, but it is very googlable